### ISSUE TITLE
New roadmap section on download singles

### ISSUE THREAD

**Issue by brashrebel** [realbigplugins/theme2016#34]
At some point I'd like to introduce a new section on the download single template which will allow for us to list out planned future features. I've seen this on other sites like [here](http://eddbookings.com/#roadmap) and customers appreciate it.

**Comment by joelworsham** [comment]
It would be cool to style it similarly to http://www.projectpanorama.com/#visualize, but vertical, and of course the content wouldn't be projects and %'s, but just future ideas, maybe that even link to GitHub issues or milestones or something. Could be neat.

**Comment by d4mation** [comment]
Only problem I can see linking to Issues/Milestones would be if the Repo were Private (Which will likely be the case 100% of the time). 

I don't think you are able to make just your Issue Tracker public on GitHub.

**Comment by joelworsham** [comment]
Well, I guess it would depend on the plugin, you're right. Well I still think the visualization woudl be nice :)

**Comment by brashrebel** [comment]
Yeah, it would be cool to make it automatic like that. But I think in the end, we'll want to make it manual. We will want to write any descriptions in a more customer appropriate way. And I would lean against progress indicators too. The purpose of this feature is to encourage a sale and providing some types of information can be counter productive. I'm envisioning some nice visualization of the feature title and possibly an icon + description.

**Comment by joelworsham** [comment]
Well, I didn't mean to take away the purpose of the example, I meant more of the layout. Like, some linear progression with various features displayed in a "fun" way.

Something like below (not exact style by ANY means, just an example)

![rbp example](https://cloud.githubusercontent.com/assets/4182573/19238682/ad9c783c-8ed0-11e6-853a-22e33d0231b1.png)

**Comment by brashrebel** [comment]
Love it! I wonder if it would make any sense to just...build from the current "features" section? Look progress down that list and the ones at the bottom will somehow be indicated that they are coming soon. Then, once they are complete, they can simply be switched to existing features.
